src/CMakeLists.txt

```diff
@@ -34,13 +39,17 @@ elseif(${CMAKE_SYSTEM_NAME} STREQUAL "Android")
     set(MVPN_PLATFORM_NAME "android")
 elseif(${CMAKE_SYSTEM_NAME} STREQUAL "iOS")
     set(MVPN_PLATFORM_NAME "ios")
+elseif(${CMAKE_SYSTEM_NAME} STREQUAL "Emscripten")
+    set(MVPN_PLATFORM_NAME "wasm")
 endif()
 
 add_compile_definitions("$<$<CONFIG:Debug>:MVPN_DEBUG>")
 add_compile_definitions("MVPN_$<UPPER_CASE:${MVPN_PLATFORM_NAME}>")
 include(cmake/${MVPN_PLATFORM_NAME}.cmake)
 
-add_subdirectory(crashreporter)
-target_link_libraries(mozillavpn PRIVATE crashreporter)
+if(NOT ${CMAKE_SYSTEM_NAME} STREQUAL "Emscripten")
```

Comment:
Would `NOT CROSS_COMPILING` be a better test here? Not that it really matters until we support Android/iOS with CMake.

Reply:
Well I hope to have crashreport on mobile eventually.

Reply:
Don't we already get crash reporting through the App/Play stores?

Reply:
not via crashpad